contrib/msmtp/files/msmtpd

```diff
@@ -0,0 +1,7 @@
+# msmtpd service
+
+type            = process
+command         = /usr/bin/msmtpd --log=syslog
+depends-on      = local.target
+depends-on      = network.target
+smooth-recovery = true
```

Comment:
```suggestion
type = process
command = /usr/bin/msmtpd --log=syslog
depends-on = local.target
depends-on = network.target
smooth-recovery = true
```

not aligned anymore

also not sure what the options might mean, but how does this compare to https://github.com/chimera-linux/cports/pull/1630 ?